src/rendering/vcAtmosphereRenderer.h

```diff
@@ -10,6 +10,9 @@ struct vcTexture;
 udResult vcAtmosphereRenderer_Create(vcAtmosphereRenderer **ppAtmosphereRenderer);
 udResult vcAtmosphereRenderer_Destroy(vcAtmosphereRenderer **ppAtmosphereRenderer);
 
+void vcAtmosphereRenderer_DestroyShaders(vcAtmosphereRenderer *pAtmosphereRenderer);
+udResult vcAtmosphereRenderer_LoadShaders(vcAtmosphereRenderer *pAtmosphereRenderer);
```

Comment:
These shouldn't need to be exposed- `Reload` is better, `Create` and `Destroy` should be handling the majority of the work.